docs/GettingStarted.md

```diff
@@ -11,21 +11,23 @@ Table of Contents
 
 ## Installation
 
-If you are using the web installer, check out this [installation page](WebInstall.md).
+Follow [these instructions](UbuntuInstall.md) to install via debian packages.
```

Comment:
Good point. I'll check and make sure that I'm consistent with using "deb package" instead of "debian package".